### ISSUE TITLE
Highlight only tab words rather than whole tab - on focus

### ISSUE THREAD

**Issue by phillipgardiner** [open-sdg/open-sdg#892]
On the gov.uk implementation of tabs: https://design-system.service.gov.uk/components/tabs/. When the tab is focused, it is only the words that are highlighted rather than the whole tab. SDG currently highlights the whole tab.

This has previously been discussed with @brockfanning as part of accessibility changes but due to time required to implement and it not being critical it was decided to create a separate ticket to implement the above.

**Comment by phillipgardiner** [comment]
Similarity with drop downs (like those used in the graph filters, only the outline of the drop down should be highlighted, not filled with yellow: https://design-system.service.gov.uk/components/select/

**Comment by brockfanning** [comment]
The tab part of this was implemented. Should we close and open a new ticket for the drop-down change?

**Comment by phillipgardiner** [comment]
@brockfanning Yes please
